pmm-app/src/pmm-dbaas/components/MultipleActions/MultipleActions.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, useState } from 'react';
+import { Icon } from '@grafana/ui';
+import { styles } from './MultipleActions.styles';
+import { MultipleActionsProps } from './MultipleActions.types';
+
+export const MultipleActions: FC<MultipleActionsProps> = ({ actions }) => {
+  const [menuOpened, openMenu] = useState(false);
+
+  const showMenu = (event) => {
+    if (menuOpened) {
+      closeMenu(event);
+
+      return;
+    }
+
+    event.preventDefault();
+    openMenu(true);
+    document.addEventListener('click', closeMenu);
+  };
+
+  const closeMenu = (event) => {
+    event.preventDefault();
+    openMenu(false);
+    document.removeEventListener('click', closeMenu);
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <button onClick={showMenu} className={menuOpened ? 'show-menu menu-open' : 'show-menu'}>
```

Comment:
Why not using classnames from emotion? If we do this can be changed to `className={cx(styles.showMenu, {[styles.showMenuOpen]: menuOpened})}`